### ISSUE TITLE
flatpickr missing

### ISSUE THREAD

**Issue by wulffeld** [matestack/matestack-ui-bootstrap#1]
Hi

Was trying out the spec/dummy demo and only had one webpacker problem:

```
ERROR in ./app/matestack/dummy/components/form/flatpickr.js
Module not found: Error: Can't resolve 'flatpickr' in '/Users/m/projects/matestack/matestack-ui-bootstrap/spec/dummy/app/matestack/dummy/components/form'
resolve 'flatpickr' in '/Users/m/projects/matestack/matestack-ui-bootstrap/spec/dummy/app/matestack/dummy/components/form'
  Parsed request is a module
  using description file: /Users/m/projects/matestack/matestack-ui-bootstrap/spec/dummy/package.json (relative path: ./app/matestack/dummy/components/form)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
```

Adding flatpickr to package.json solved the problem for me.

**Comment by jonasjabari** [comment]
@wulffeld Thanks for pointing this out! Will be fixed within the next days! Btw: We're working on 2.0 of matestack-ui-core and matestack-ui-bootstrap currently. Will be release beginning of April! If you have any further questions, don't hesitate to jump on our Discord server and discuss your feedback/issues :) Cheers!

**Comment by jonasjabari** [comment]
This issue was fixed within the 2.0.0 release
